### DISCUSSION TITLE
importing helper_functions

### DISCUSSION THREAD

**Discussion by DavS123** [mrdbourke/pytorch-deep-learning#207] [Q&A]
Hello everyone,

for some reason I get import error when I am trying to import the accuracy_fn function from the helper_function file when I am using "none" accelerator mode in Google Colab. However when I am using the GPU mode everything works fine. Can some please explain why that is 

best wishes

**Comment by mrdbourke** [comment]
Hi @DavS123,

I'm not sure what's going on here, I've got no problems importing and using the helper functions.

Have you tired downloading the file manually and uploading it to Google Colab?

Or even just running the whole contents of `helper_functions.py` in a single Colab cell? (so you don't have to import from `helper_functions`)
